.github/workflows/consistency-check.yaml

```diff
@@ -30,6 +30,10 @@ jobs:
       - name: Verify Codegen
         working-directory: ./ray-operator
         run: ./hack/verify-codegen.sh
+
+      - name: Verify Generated Files
```

Comment:
Just a nit-picky comment, we can have a better name here. 

-`Verify Generated Files`: showing it is used to verify the [aforementioned runtime.Object interface](https://book.kubebuilder.io/cronjob-tutorial/other-api-files.html) generated by kubebuilder. 

-`Verify Codegen`: showing it is used to verify things like clientset, lister, informer

Reply:
I got the idea from Kubernetes repo (https://github.com/kubernetes/kubernetes/blob/v1.22.1/hack/verify-generated-files.sh). Please suggest better alternatives.